### DISCUSSION TITLE
include multiple files from a directory

### DISCUSSION THREAD

**Discussion by cosoc** [tauri-apps/tauri#5884] [Q&A]
I want to include directory resources
![image](https://user-images.githubusercontent.com/19447682/208850505-8ba90362-b52a-4f1b-89ba-3e5033764521.png)

erro
![image](https://user-images.githubusercontent.com/19447682/208850804-f31a36cf-e463-428a-92b2-e1b78d1f8614.png)

**Comment by FabianLars** [comment]
In _theory_ that should work. A "permission denied" error is weird tho and should not be cause by the config and should be actually a permission issue.
But just to be sure, can you check if `extends/*` and `extends/**` don't work either please?

**Reply by cosoc** [reply]
> In _theory_ that should work. A "permission denied" error is weird tho and should not be cause by the config and should be actually a permission issue. But just to be sure, can you check if `extends/*` and `extends/**` don't work either please?

I've tried it all. The absolute path is OK, but the relative path is not!
